authorization.md

```diff
@@ -1,29 +1,37 @@
 # SMART Backend Services: Authorization Guide
 
   
-## Profile audience and scope
+## Profile Audience and Scope
 
 This profile is intended to be used by developers of back-end services that
-need to access FHIR resources by requesting access tokens from OAuth 2.0
-compliant authorization servers. This profile assumes that a backend service
-has been authorized up-front, at registration time, and describes the runtime
-process by which the service acquires an access token that can be used to
-communicate with a FHIR Resoure Server.
+autonomously (or semi-autonomously) need to access FHIR resources from an EHR 
+that has pre-authorized a defined scope of access.  Specifically, this profile 
+describes the registration process by which a backend service is pre-authorized, 
```

Comment:
It does not really describe the registration process, does it?

Reply:
It describes functional requirements for the process; would you prefer different wording here?

Reply:
I prefer the old wording `This profile assumes that a backend service
has been authorized up-front`

Reply:
OK. @bakerdb perhaps we can use a hybrid like:

```suggestion
describes the registration-time metadata required for a backend service to be pre-authorized,
```

Reply:
I don't think we provide much detail regarding the registration process at all.   We just require that the backend client has pre-registered and that the EHR has authorized a scope of access.

Reply:
I like your suggested wording here, Josh.

Reply:
Cool  -- I think you can click to accept the proposed change (this is a new feature in GitHub; if you don't see it, just let me know and I'll apply directly).

Reply:
I see the "Apply Suggestion" button.  But when I click it, it says "Outdated Suggestions cannot be applied."  Guess I took too long.

Reply:
Oh, ok -- you successfully accepted the suggestion yesterday, and it's now reflected in this PR. So... good :-)